scripts/ebcli_installer.py

```diff
@@ -139,15 +139,17 @@ def _exec_cmd(args):
 
 
 INSTALLATION_SUCCESS_MESSAGE_WITH_EXPORT_RECOMMENDATION__NON_WINDOWS = """
-    Note: To complete installation, ensure `eb` is in PATH. You can ensure this by executing:
+    Note: To complete installation, ensure `eb` and 'python' are in PATH. You can ensure this by executing:
```

Comment:
Repetition of `ensure` is awkward. Maybe you can combine the two To complete installation, verify that `eb` and `python` are in your path by running one of the following commands:

Reply:
I think that's cleaner, will update soon.

Reply:
A note on the update I made, I changed the wording to `To complete installation, verify that `eb` and `python` are in PATH by running the following commands:` because depending on someone's OS they will need to run two echo commands. The wording with `one of the following commands` I think may have given the impression that someone only needed to run one of them.